command.php

```diff
@@ -15,64 +15,78 @@
 
 /**
  * WP-CLI PO command for satellitewp/po
+ *
+ *  ## EXAMPLES
+ *
+ *      # Merge for basic usage.
+ *      $ wp po merge fr-ca fr https://wordpress.org/plugins/wordpress-seo/
+ *
+ *      # Merge for Custom locale.
```

Comment:
I'd suggest: Merge for the translation URL of a project in the WordPress repository (including meta and apps)